doc/nrf/releases_and_maturity/migration/migration_guide_3.0.rst

```diff
@@ -105,16 +105,16 @@ ZMS settings backend
    To migrate from the legacy backend to the new backend remove the Kconfig options :kconfig:option:`CONFIG_SETTINGS_ZMS_NAME_CACHE`
    and :kconfig:option:`CONFIG_SETTINGS_ZMS_NAME_CACHE_SIZE` from your conf files.
 
-Bluetooth LE legacy pairing
----------------------------
+Bluetooth® LE legacy pairing
+----------------------------
 
 .. toggle::
 
-   Support for Bluetooth LE legacy pairing is no longer enabled by default, because it is not secure.
+   Support for Bluetooth LE legacy pairing (:kconfig:option:`CONFIG_BT_SMP_SC_PAIR_ONLY`) is no longer enabled by default because it is not secure.
```

Comment:
I would skip mentioning the Kconfig option name here as it uses inverted logic (enabling it disables the BLE legacy pairing support). Also the option is already linked in the subsequent line.